shell/browser/electron_download_manager_delegate.h

```diff
@@ -60,6 +67,17 @@ class ElectronDownloadManagerDelegate
 
   content::DownloadManager* download_manager_;
   base::WeakPtrFactory<ElectronDownloadManagerDelegate> weak_ptr_factory_{this};
+
+#if BUILDFLAG(IS_WIN)
+  // Copied from ui/shell_dialogs/select_file_dialog_win.h
+  bool GetRegistryDescriptionFromExtension(const std::u16string& file_ext,
+                                           std::u16string* reg_description);
+  std::vector<ui::FileFilterSpec> FormatFilterForExtensions(
```

Comment:
For utility functions that are only used inside one file, they should usually be put in the anonymous namespace in the `.cc` file. For example the original `FormatFilterForExtensions` was in the anonymous namespace of the `select_file_dialog_win.cc` instead of being a method of `SelectFileDialogImpl`.

Reply:
I see! That makes sense. Thank you